olcPGEX_AdditionalColours.h

```diff
@@ -60,5 +60,60 @@ namespace olc
 			PALE_DARK_COBALT(75, 114, 133),
 			PALE_DARK_PURPLE(83, 69, 133),
 			PALE_DARK_PINK(130, 66, 133);
+
+		// Taken from https://en.wikipedia.org/wiki/HSL_and_HSV#HSV_to_RGB
+		// Creates a new Pixel instance from hsv space (hue, saturation and value).
+		// Hue ranges from 0 to 360, saturation and value from 0 to 1.
+		static olc::Pixel FromHsv(float hue, float saturation, float value, const float alpha = 1)
+		{
+			hue = std::clamp(hue, 0.0f, 360.0f);
+			saturation = std::clamp(saturation, 0.0f, 1.0f);
+			value = std::clamp(value, 0.0f, 1.0f);
+			
+			const float chroma = value * saturation;
+			const float x = chroma * (1 - std::fabs(fmodf((hue / 60), 2) - 1));
```

Comment:
What are `x` and `m`? Probably worth giving them more descriptive names.

Reply:
x appears to be the second largest component of the color, m is used to match the raw colors to the given value. Most implementations and writeups don't give them better names either.